UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs

```diff
@@ -975,12 +975,12 @@ public void UpdateLocalAndWorldConversion()
 	public void UpdateSyncVars()
 	{
 		if (isServer == false) return;
-		if (SynchronisedSpin != CurrentTorque)
+		if (Mathf.Approximately(SynchronisedSpin, CurrentTorque))
```

Comment:
shouldn't you negate this? Since the original code is a not comparison

Reply:
fixed